app/routes/bps-multiple.js

```diff
@@ -21,7 +21,24 @@ module.exports = [{
         bps2022Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000),
         bps2023Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000),
         bps2024Value: joi.number().empty('').allow(null).precision(2).greater(0).less(1000000000)
-      }).min(1),
+      }).min(1).error(errors => {
+        errors.forEach(err => {
+          switch (err.code) {
+            case 'object.min':
+              err.message = 'There needs to be a least one payment amount for a scheme year.'
```

Comment:
There should be a rule for not entering a valid number then I'd say anything else can be in the catch all mentioned below